lib/labels/github/api.ex

```diff
@@ -8,4 +8,42 @@ defmodule Labels.Github.Api do
 
   @impl true
   def get_profile(code), do: ElixirAuthGithub.github_auth(code)
+
+  @impl true
+  def get_labels(token, owner, repo) do
+    req = Req.new(base_url: "https://api.github.com")
+
+    res =
+      Req.get!(req, url: "/repos/#{owner}/#{repo}/labels?per_page=100", auth: {:bearer, token})
+
+    case res.status do
+      200 -> {:ok, res.body}
+      404 -> {:error, :not_found}
+      _ -> {:error, :unknown}
+    end
+  end
+
+  @impl true
+  def create_label(token, owner, repo, label) do
+    res =
+      Req.post!("https://api.github.com/repos/#{owner}/#{repo}/labels",
+        auth: {:bearer, token},
+        json: label
+      )
+
+    {:ok, res.status}
+  end
+
+  @impl true
+  def update_label(token, owner, repo, label_name, label) do
+    endpoint = URI.encode("https://api.github.com/repos/#{owner}/#{repo}/labels/#{label_name}")
```

Comment:
Uses `URI.encode` to make sure the endpoint is valid when the <label_name> contains space (or other specific characters), for example "help wanted"